### ISSUE TITLE
OpenAPI 3.0 schema validation runs only once

### ISSUE THREAD

**Issue by apozeus** [a85/POSTMan-Chrome-Extension#10411]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Upon making collection request, schema validation issues only show on first run. Subsequent requests will not show any issue.
Need to restart postman for issues to show again.

### Steps To Reproduce

1. Generate collection with API definition (OpenAPI 3.0, yaml).
2. Run request
3. Validation issues are shown
4. Run request again
5. No validation issues are shown.
6. Restart postman app
7. Run request again
8. Validation issues are shown.

### Screenshots or Videos

<img width="490" alt="2021-10-30_08-49-51" src="https://user-images.githubusercontent.com/6308465/139514477-0ef2ccba-9b2a-401b-8127-b66ea54a0914.png">

### Environment Information

```markdown
- Operating System: MacOS (64)
- Platform Type: Native App
- Postman Version: 9.1.3
```

**Comment by apozeus** [comment]
Postman 9.1.1 does not have this issue.

**Comment by akshaydeo** [comment]
Hello @apozeus, thanks for reporting this issue. We are looking into it 👍

**Comment by hrhv** [comment]
Hi @apozeus, thanks for bringing this issue in light.

We've fixed this as of `v9.7.0` but recommend using the latest version to have the best experience with Postman.
You can get the latest version from [here](https://www.postman.com/downloads/).

Please feel free to reach out in case of any issues / concerns.

Closing this! cc: @akshaydeo

**Comment by weskoerber** [comment]
Hello, I'm experiencing this same issue. Schema validation runs for a single request. After that, it doesn't work, it just keeps displaying the warnings from the previous request. When I close the window and reopen it, it seems to work for exactly one request again, then subsequent requests the old behavior returns.

The issue is present in both Flatpak Postman and Postman in the browser. Flatpak Postman version 9.13.2 and Postman browser version 9.14.0-220214-1200

I'm on Fedora 35.

Edit: Help > Clear Cache and Reload makes it work for one request again then it stops working after the first request.
